website/demos/Rtl.tsx

```diff
@@ -0,0 +1,32 @@
+import DataGrid from '../../src';
+import type { Column, FormatterProps } from '../../src';
+
+type Row = number;
+const rows: readonly Row[] = [...Array(100).keys()];
+
+function CellFormatter(props: FormatterProps<Row>) {
+  return (
+    <>
+      {props.column.key}&times;{props.row}
+    </>
+  );
+}
+
+const columns: Column<Row>[] = [];
+
+for (let i = 0; i < 50; i++) {
+  const key = String(i);
+  columns.push({
+    key,
+    name: key,
+    frozen: i < 5,
+    resizable: true,
+    formatter: CellFormatter
+  });
+}
+
+export default function Rtl() {
```

Comment:
Would be nice to have a lrt/rtl toggle to check whether we correctly handle direction changes.

Reply:
Should I remove this example and add a global toggle?

Reply:
I'd keep this example IMO